src/test/java/com/testCases/UpdateUserDetails_Tcs.java

```diff
@@ -2,34 +2,130 @@
 
 import Base.Utilities;
 import com.commonMethods.AllureLogger;
+import io.restassured.RestAssured;
 import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
 import org.testng.annotations.Test;
 
 import static io.restassured.RestAssured.given;
 
 public class UpdateUserDetails_Tcs extends Utilities {
 
-    String payload = "[\n" +
-            "  {\n" +
-            "    \"id\": 1,\n" +
-            "    \"username\": \"Pulkit\",\n" +
-            "    \"firstName\": \"Nitesh\",\n" +
-            "    \"lastName\": \"Dhiman\",\n" +
-            "    \"email\": \"Nitesh@gmail.com\",\n" +
-            "    \"password\": \"Test@1234\",\n" +
-            "    \"phone\": \"99965222234\",\n" +
-            "    \"userStatus\": 0\n" +
-            "  }\n" +
-            "]\n";
-
-    @Test(priority = 1, description = "Update user list of users with given array", enabled = true)
-    public void update_UserList() {
+    String payload = "{ \"id\": 568, \"username\": \"Jimmy\", \"firstName\": \"John\", \"lastName\": \"Doe\", \"email\": \"john.doe@example.com\", \"password\": \"password123\", \"phone\": \"1234567890\", \"userStatus\": 1 }";
+
+    String invalid_PhoneNumber_Payload = "{ \"id\": 568, \"username\": \"Jimmy\", \"firstName\": \"John\", \"lastName\": \"Doe\", \"email\": \"john.doe@example.com\", \"password\": \"password123\", \"phone\": \"12345621576241761567890\", \"userStatus\": 1 }";
```

Comment:
Use Request SPecification file donot hard code api request ( change it for all apis)